libs/components/validation/src/lib/modules/validation/validation.ts

```diff
@@ -8,8 +12,39 @@ export abstract class SkyValidation {
     return regex.test(emailAddress);
   }
 
-  public static isUrl(url: string): boolean {
-    const regex = /^((http|https):\/\/)?([\w-]+\.)+[\w-]+/i;
-    return regex.test(url);
+  public static isUrl(
+    url: string,
+    urlValidationOptions?: SkyURLValidationOptions
+  ): boolean {
+    console.log(
+      'skyUrlValidationOptions in Validation.ts function',
+      urlValidationOptions
+    );
+    if (urlValidationOptions && urlValidationOptions.rulesetVersion === 2) {
```

Comment:
fixed in commit: [use switch statement rather than if. add new tests for 100% code cove…](https://github.com/blackbaud/skyux/pull/201/commits/9298cb3ca626ef78b73e70c79e322fa54abe8d49)